src/main/java/frc/robot/commands/StopDrive.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+public class StopDrive extends CommandBase {
+  /** Creates a new StopDrive. */
+  private final Drivetrain drivetrain;
+  
+  public StopDrive(Drivetrain drivetrain) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    this.drivetrain = drivetrain;
+    addRequirements(drivetrain);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
```

Comment:
Do not include empty overrides unless the `super()` method performs unwanted actions